README.md

```diff
@@ -56,6 +56,22 @@ and health-checking mechanisms.
   ```sh
   sudo yum localinstall -y https://yum.puppetlabs.com/puppet/el/7/x86_64/puppet-agent-7.0.0-1.el7.x86_64.rpm
   ```
+   * **Note:**   
+     Check facter version details, if already installed / exist in system
+     ```sh
+     facter -v
+     ls -ll /usr/bin/facter
+     ```
+     Supported version is **facter >= 3.14.8**, If the facter version is **< 3.14.8** then follow the steps below: 
+     ```sh
+     rm -rf /usr/bin/facter
+     yum list installed | grep "puppet-agent"
```

Comment:
A cleaner way would be to learn which package owns `/usr/bin/facter` and remove that one if it exists (some users may install facter just from sources - in that case your `rm -f /usr/bin/facter` makes perfect sense).

This command finds the installed package that has brought the particular file: `rpm -q --whatprovides [file name]`

So I can suggest the following command:
```
# yum erase -y $(rpm -q --whatprovides /usr/bin/facter) || rm -fv /usr/bin/facter
```